features/org.wso2.carbon.event.handler.notification.server.feature/resources/SMSPublisher.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we have the capability to configure this via the deployment.toml?

Reply:
Improvement is tracked in https://github.com/wso2/product-is/issues/10859